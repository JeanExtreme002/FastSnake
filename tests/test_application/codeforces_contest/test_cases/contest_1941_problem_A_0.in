4
4 4 8
1 5 10 14
2 1 8 1
2 3 4
4 8
1 2 3
4 2 7
1 1 1 1
2 7
3 4 2000
1 1 1
1 1 1 1