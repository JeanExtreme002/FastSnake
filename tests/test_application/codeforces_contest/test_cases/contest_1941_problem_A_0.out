6
0
4
12